scripts/qiita

```diff
@@ -384,13 +386,20 @@ def unlock():
               help='The amount of time to show message, in seconds')
 @click.option('--message', required=True, type=str,
               help="Message to show users")
-def sysmessage(message, time):
-    r_client.setex('sysmessage', message, time)
+@click.option('--banner', is_flag=True,
+              help="If set, show as site banner instead of a standard message")
+def sysmessage(message, time, banner):
```

Comment:
This may be a discussion, but it is well outside the already ever-increasing scope of features in this pull request. To keep it from getting any bigger, and to make the deadline, please open an issue and we can discuss there.